tasks/run-matlab-build/v0/test/failRunBuild.ts

```diff
@@ -14,6 +14,10 @@ tr.registerMock("./utils", {
     architecture: () => "x64",
 });
 
+tr.registerMock("fs", {
+    chmodSync: () => Promise.resolve(0),
+});
```

Comment:
Huh, was this failing before? Or did it just work even with that not mocked out?

Reply:
It was working. It was just operating on the actual file. This time the file didn't exist (because it needs to be unzipped) so it was failing. I wanted to make sure our tests don't ever actually unzip anything so we don't accidentally ship both the zip and the unzipped binary